backend/experiment/views.py

```diff
@@ -14,13 +17,21 @@
 )
 from experiment.rules import BLOCK_RULES
 from experiment.actions.utils import EXPERIMENT_KEY
+from experiment.serializers import ExperimentSerializer
+
 from participant.models import Participant
 from participant.utils import get_participant
 from theme.serializers import serialize_theme
 
 logger = logging.getLogger(__name__)
 
 
+class ExperimentViewSet(viewsets.ModelViewSet):
+    queryset = Experiment.objects.all()
+    serializer_class = ExperimentSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+
```

Comment:
This creates GET, GET(id), POST, PUT & DELETE endpoints for the experiment (and its relations as defined in the serializers)